From fc6ceff76f24cf433ada3013dfcb944880326b49 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Tue, 7 Mar 2017 15:36:17 -0500
Subject: [PATCH 94/94] ti_config_fragments: add jailhouse config overlay

* For the jailhouse hypervisor to function, it requires certain config
  options to be enable, such as CONFIG_HOTPLUG_CPU.
* Add a seperate config overlay in case there are more requirements in
  the future.
* In addition, add new "jailhouse" targets to the defconfig_map.txt to
  preserve the existing defconfig targets.

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 ti_config_fragments/defconfig_map.txt | 3 +++
 ti_config_fragments/jailhouse.cfg     | 5 +++++
 2 files changed, 8 insertions(+)
 create mode 100644 ti_config_fragments/jailhouse.cfg

diff --git a/ti_config_fragments/defconfig_map.txt b/ti_config_fragments/defconfig_map.txt
index 5a0ed65..8a34f8f 100644
--- a/ti_config_fragments/defconfig_map.txt
+++ b/ti_config_fragments/defconfig_map.txt
@@ -29,3 +29,6 @@ classification: SDK_Debug_Defconfigs type: ti_sdk_keystone_rt_debug defconfig: m
 # Android Defconfigs
 classification: SDK_Release_Defconfigs type: ti_sdk_dra7x_android_release defconfig: multi_v7_defconfig config_file: None extra_configs: multi_v7_prune.cfg baseport.cfg ipc.cfg connectivity.cfg audio_display.cfg wlan.cfg omap_soc.cfg lpae.cfg dra7_only.cfg systemd.cfg ../android/configs/android-base.cfg ../android/configs/android-recommended.cfg
 classification: SDK_Debug_Defconfigs type: ti_sdk_dra7x_android_debug defconfig: multi_v7_defconfig config_file: None extra_configs: multi_v7_prune.cfg baseport.cfg ipc.cfg connectivity.cfg audio_display.cfg wlan.cfg omap_soc.cfg lpae.cfg dra7_only.cfg systemd.cfg debug_options.cfg ../android/configs/android-base.cfg ../android/configs/android-recommended.cfg
+# Jailhouse Support Defconfigs
+classification: SDK_Release_Defconfigs type: ti_sdk_dra7x_jailhouse_release defconfig: multi_v7_defconfig config_file: None extra_configs: multi_v7_prune.cfg baseport.cfg ipc.cfg connectivity.cfg audio_display.cfg wlan.cfg omap_soc.cfg lpae.cfg dra7_only.cfg systemd.cfg jailhouse.cfg
+classification: SDK_Release_Defconfigs type: ti_sdk_dra7x_rt_jailhouse_release defconfig: multi_v7_defconfig config_file: None extra_configs: multi_v7_prune.cfg baseport.cfg ipc.cfg connectivity.cfg audio_display.cfg wlan.cfg omap_soc.cfg lpae.cfg dra7_only.cfg real_time.cfg systemd.cfg jailhouse.cfg
diff --git a/ti_config_fragments/jailhouse.cfg b/ti_config_fragments/jailhouse.cfg
new file mode 100644
index 0000000..4fba210
--- /dev/null
+++ b/ti_config_fragments/jailhouse.cfg
@@ -0,0 +1,5 @@
+##################################################
+# Config required to run Jailhouse Hypervisor
+##################################################
+
+CONFIG_HOTPLUG_CPU=y
-- 
1.9.1

